client/src/api/datasource/local-client.ts

```diff
@@ -44,7 +47,8 @@ export class LocalClient implements DataSourceClient {
 
   features() {
     return {
-      update_meta: false,
+      updateMeta: false,
```

Comment:
Seems like you're doing the reverse of this naming in other files?

Reply:
JavaScript it is camelCase python sneak_case